extensions/amp-mustache/0.1/test/test-amp-mustache.js

```diff
@@ -319,11 +318,13 @@ describe('amp-mustache template', () => {
       const nestedTemplateElement = outerResult.querySelector('template');
       const nestedTemplate = new AmpMustache(nestedTemplateElement);
       nestedTemplate.compileCallback();
-      const nestedResult = nestedTemplate.render({
-        value: 'Nested',
-      });
-      expect(nestedResult./*OK*/innerHTML).to.equal(
-          '<div>nested</div>: Nested');
+      expect(() => {
+        const nestedResult = nestedTemplate.render({
+          value: 'Nested',
+        });
+        expect(nestedResult./*OK*/innerHTML).to.equal(
```

Comment:
Curious: If `render()` throws an error, does this follow up `expect` get run at all? Will the test fail if you put in something like `expect(false).to.equal(true)` after this line?

Reply:
Oops, you're right. I reverted the test and made the sanitizer throw warnings, not errors, since these are actual genuine warnings and not errors